suites/quincy/rbd/tier-3_rbd_snap_operations.yaml

```diff
@@ -97,5 +98,5 @@ tests:
             data_pool: rbd_ec_pool
             rbd_ec_scale_snap_image:
               size: 1G
-              num_of_snaps: 5
+              num_of_snaps: 500
```

Comment:
we can, though in separate testcase like clone focussed scale test, as in this case it will blow out of proportions Eg 500 snapshots and lets say 500 clone per snapshots, its massive :)